docs/development.rst

```diff
@@ -481,6 +481,54 @@ QQ-plots of the statistics in question comparing ``ms`` and ``msprime``.
 There are also several "analytical" tests, which compare the distributions of
 values from ``msprime`` with analytical expectations.
 
+****************
+Containerization
+****************
+
+This repo is integrated with Dockerhub and the Docker image will be automatically
```

Comment:
Maybe some links here? Perhaps a link to the Dockerhub page for ``tskit/msprime`` somewhere?